ci-operator/jobs/openshift/sriov-network-operator/openshift-sriov-network-operator-master_okd-scos-presubmits.yaml

```diff
@@ -0,0 +1,58 @@
+presubmits:
+  openshift/sriov-network-operator:
+  - agent: kubernetes
+    always_run: true
```

Comment:
you want this to be false and `optional: true`

Reply:
I can see in the multus example you point out it's the same
https://github.com/openshift/release/pull/50439

and I want this one to run on every merge to master